src/Entity/Domain/AuthorizedDomain.php

```diff
@@ -8,6 +8,7 @@
 use Sylius\Component\Resource\Model\ResourceInterface;
 use Synolia\SyliusAdminOauthPlugin\Repository\AuthorizedDomainRepository;
 
+#[ORM\Table(name: 'synolia_authorized_domain')]
```

Comment:
Isn't it a good idea to do this for all attributes? For example, `$isEnabled` will be `isEnabled` in the database, won't it?

Reply:
Totally